stories/topics/con-saas-intro.adoc

```diff
@@ -7,5 +7,5 @@ From here you can access the control plane which allows you to run and manage yo
 You can connect execution nodes through an egress model from your environment using mesh ingress.
 Alternatively, mesh ingress sets up hop nodes on the controller and then you can peer execution nodes to them.
 
-
-//Insert diagram here when ready DXD-716
+.Figure 1. Ingress vs Egress
+image:ingress_vs_egress.png[Ingress vs Engress] 
```

Comment:
I think a typo here, should `Engress` be `Egress`?

Reply:
Indeed it should! Good spot!